### PR TITLE
Fix/symlink

### PR DESCRIPTION
**Pull request by ahelal**

### CODE REVIEW

File: package-scripts/kitchen/postinst [hunk 1]
```diff
@@ -11,7 +11,7 @@ error_exit()
   echo "${PROGNAME}: ${1:-"Unknown Error"}" 1>&2
   exit 1
 }
-ln -s /usr/local/bin/kitchen /opt/hellofresh/bin/kitchen
+ln -s /opt/hellofresh/bin/kitchen /opt/hellofresh/bin/kitchen
```
Comment (zeelax) [inline package-scripts/kitchen/postinst]: Did you mean another path?

Comment (ahelal) [inline package-scripts/kitchen/postinst]: That's correct all hellofresh tools will leave on /opt/hellofresh/bin

**Comment by ahelal** [comment]
@zeelax fixed

**Comment by zeelax** [comment]
+1

**Pull request merged**